packages/core/src/components/icon/icon.story.tsx

```diff
@@ -1,14 +1,12 @@
-import { c, classy, color, IconProps as BaseProps } from '@onfido/castor';
+import { c, classy, color, IconProps } from '@onfido/castor';
 import { html } from '../../../../../docs';
 
-export interface IconProps extends BaseProps {
-  'aria-hidden'?: 'true' | 'false';
-}
-
 /**
  * `.ods-icon` requires SVG sprite to be included in your app.
  *
  * https://github.com/onfido/castor-icons
+ *
+ * Requires either `aria-hidden` or `aria-label` for accessibility.
```

Comment:
Should it be:

```suggestion
 * Requires either `aria-hidden="true"` or `aria-label` for accessibility.
```

What do you think?

Reply:
The value is going to be defined by the type, which actually makes me realise it's wrong for non-react, can't be a boolean.

I was just listing the name of the attributes in the comment, it does make sense to show an example, but then should it be `aria-label=""` too?

Reply:
Well no, because for `aria-hidden` a requirement is specifically for it to be set as `"true"`, but not `"false"`.

For `aria-label` value does not matter, as long as it is set.

Reply:
Fair enough